### ISSUE TITLE
gold-phone-input accessibility issues

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/gold-phone-input#6]
Focus animation is not consistent across input fields, so keyboard focus is not visible. Please ensure the animation is consistent and that it is clear where focus is. 

Contrast: The hint text and "phone number" text is too light. 

Audit results: 
[Severe] Controls and media elements should have labels (4)
[Warning] Text elements should have a reasonable contrast ratio (3)
Not applicable tests (9)
Passing tests (6)

**Comment by notwaldorf** [comment]
Done! :tada:
